src/frontend/apps/bbb/SelectContentTab/index.tsx

```diff
@@ -153,10 +151,6 @@ export const SelectContentSection = ({
   items,
   selectContent,
 }: SelectContentSectionProps) => {
-  if (!isFeatureEnabled(flags.BBB)) {
-    return null;
-  }
-
```

Comment:
I removed this check because parent component already load the component only if the flag is enabled and I don't think we should check the same think several times at multiples locations.

Note : this did not raise any error in tests, there is eventually a lack of test checking this component can only be load if the feature flag is enabled